day2.md

```diff
@@ -0,0 +1,171 @@
+
+Walking around the market and surrounding by foods, especially fresh seafoods, got me very excited. The 5-years-old inside me had been screaming, jumping, and dancing all around the whole time as we were walking through the main street and deciding where we should have our first meal of the day. Many of you should be familiar with the analogy that life sometimes is like walking through a forest knowing that you can only pick one fruit from a tree without knowing what's ahead of you nor the opportunity of returning and picking from a tree you have passed. That's exactly how I felt when we were in the fish market and any other times we had to make a decision related to food in Japan! We could only ate so much each day and there were so many yummies surrounding and calling us. As a computer science major, I couldn't help but tried to collect as many relevant data as possible and run it through my real-time "where-should-we-eat" algorithm<sup>[1](#algorithm)</sup>.
+
+I have to apologize that we didn't take many great pictures in the fish market because it was fairly crowded and all the foods there were screaming for our attentions.
+
+<img src="day2/IMG_20160906_101301.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_102331.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_102433.jpg" width="500"><br/>
+Here comes the first thing I got: my all time favorite tamagoyaki（玉子焼き）！
+
+<img src="day2/IMG_20160906_103227.jpg" width="500"><br/>
+<img src="day2/14409662_1115752155141144_7886916382605174951_o.jpg" width="500"><br/>
+<img src="day2/14444831_1115752548474438_2131506267391325625_o.jpg" width="500"><br/>
+<img src="day2/14435156_1115752615141098_910067847601582150_o.jpg" width="500"><br/>
+Then comes scallop sautéed in butter.
+
+<img src="day2/14409862_1115752071807819_502330450151416461_o.jpg" width="500"><br/>
+There are couple small sanding tables in the shop and we just ate there right after it was cooked!<br/>
+I wished I had tried out those fresh sashimi there. I saw someone had tuna cubes with the middle parts still icy.
+
+After enough wondering and calculation, we picked<sup>[2](#pick)</sup> a small sushi shop and ate there.
+<img src="day2/IMG_20160906_110423.jpg" width="500"><br/>
+The "restaurant" was very small. We sat on the other side of it, but that's pretty much it. Most "restaurants" in the midst of the market were about the same size. For some reason, my brain just connect these type of shops with adjectives such as "authentic" or "local"<sup>[3](#small)</sup>.
+
+<img src="day2/IMG_20160906_104720.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_105812.jpg" width="500"><br/>
+Here comes our sashimi bowel and salmon sashimi!<br/>
+They were good but a bit pricy and didn't live up to our expectations.
+
+<img src="day2/IMG_20160906_111430.jpg" width="500"><br/>
+We also grabbed some of these before leaving the market!😄<br/>
+One of my favorite snacks in Japan, I miss them!
```

Comment:
Technically, this comma would be incorrect for formal English (replace with semicolon or period since these are two independent clauses), but I think they are perfectly acceptable in informal things like blog posts... Honestly, I think you can just leave this. I'm just babbling now.